### PR TITLE
Clarify docs for get_* methods

### LINKED ISSUE #131: get_* methods returning single dict on single-object returns (should be dict within list)
When it's provided with a username, it should return a single dict. Instead, it returns a list containing a single dict.

This should be a simple fix, I'll create a PR in a sec.

### PR DESCRIPTION
**Pull request by matthewrsj**
Fixes #131
It is now clear that the get_* methods return a list containing
zero or more python dicts.

### CODE REVIEW

**Comment by matthewrsj** [comment]
@subnomo @aaroncohen73 @Kennric

**Comment by aaroncohen73** [comment]
You say "even if the list contains zero or one" for get_times(), but you instead say "even if the list contains one" for all the other get_* methods.

**Comment by matthewrsj** [comment]
@aaroncohen73 that's because the other methods should return an error instead of an empty list. get_times is the only one that allows you to supply *filter* parameters.

**Comment by aaroncohen73** [comment]
@matthewrsj Oh yeah, forgot about that. That aside, this gets my +1

**Comment by matthewrsj** [comment]
@subnomo @Kennric please review. This **should** be the behavior of pymesync. The fact that the behavior is contrary is not the docs fault and #131 should be fixed separately.

File: docs/source/usage.rst [hunk 1]
```diff
@@ -355,7 +355,8 @@ TimeSync.\ **get_times(query_parameters=None)**
     provided when instantiating the TimeSync object. The time entries are
     filtered by parameters passed in ``query_parameters``. Returns a list of
     python dictionaries containing the time information returned by TimeSync or
-    an error message if unsuccessful.
+    an error message if unsuccessful. This method always returns a list, even
+    if the list contains zero or one time object.
```
Comment (ElijahCaine) [inline docs/source/usage.rst]: Is a `one time object` a single object?

**Comment by ElijahCaine** [comment]
+1

File: docs/source/usage.rst (hunk 1 above)
Comment (matthewrsj) [inline docs/source/usage.rst]: yes

**Pull request merged**